config/locales/hyrax.en.yml

```diff
@@ -736,6 +736,7 @@ en:
       no_transfers: You haven't transferred any work
       proxy_activity: Proxy Activity
       proxy_delete: Delete Proxy
+      proxy_help: Select a user who can deposit works on your behalf. You will be the owner of works that this user deposits for you. You can revoke the proxy by clicking the Remove button beside the proxy.
```

Comment:
With the button text reading Delete proxy, proxy_help needs to change to read 'clicking the Delete proxy button beside the proxy'.